setup.py

```diff
@@ -2,23 +2,29 @@
 from setuptools import setup
 from setuptools import find_packages
 
+LONG_DESCRIPTION = ""
 with open("README.md", encoding="utf-8") as readme_file:
-    long_description = readme_file.read()
+    LONG_DESCRIPTION = readme_file.read()
 
-about = {}
+ABOUT = {}
 with open(pathlib.Path("pycytominer/__about__.py")) as fp:
-    exec(fp.read(), about)
-print(about)
+    exec(fp.read(), ABOUT)
+
+# pull requirements for install_requires
+REQUIRED_PKGS = []
+with open('requirements.txt') as f:
```

Comment:
Please make sure to `black` this file too! :) (I notice this line b/c of single quotes)

Reply:
Will do, apologies about this.

Reply:
No worries!